config/routes.rb

```diff
@@ -20,6 +20,8 @@
   get '/measurements/electrical/new', to:'measurements#new_electrical'
   get '/measurements/internal_conditions/new', to:'measurements#new_internal_conditions'
   get '/measurements/meteorological/new', to:'measurements#new_meteorological'
+  get '/measurements/wind_turbine_speed/new', to:'measurements#new_wind_turbine_speed'
```

Comment:
Speed is the only field of the WindTurbineSeedMeasurement model. Could we use only one model/table called WindTurbineMeasurement and include there all the fields related to the WindTurbine?